libs/sdk-core/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # 1.5.0 (unreleased)
 
 ### Breaking changes
+
 - Remove `Search.SmartParagraph` interface as it's not used anymore
 - Rename `Search.SmartResult` interface to `Search.FieldResult` as `Smart` prefix doesn't make sense anymore
+- Add `selections` in field metadata
```

Comment:
doesn't seem like a breaking change to me

Reply:
I'll fix it later, I need to deploy